src/components/MultiSelect/MultiSelect.mdx

```diff
@@ -27,86 +28,142 @@ import Link from "../Link/Link";
 
 <Description of={MultiSelectStories} />
 
-## Component Props for MultiSelect Listbox
+## Component Props
 
-The `MultiSelect` listbox renders a non-hierarchical list of items to select from.
+The `MultiSelect` renders a non-hierarchical list of items to select from.
 
-<Canvas of={MultiSelectStories.ListboxControls} />
+<Canvas of={MultiSelectStories.withControls} />
 
-<ArgTypes of={MultiSelectStories.ListboxControls} />
-
-## Component Props for MultiSelect Dialog
-
-The `MultiSelect` dialog allows for an optional set of child checkboxes to be
-passed, which makes the “parent” checkbox function as a check/uncheck all toggle.
-If all of the children checkboxes are checked, the parent `isChecked` prop will
-be true. If only some of the child checkboxes are checked, the parent checkbox
-will have a `isIndeterminate` prop set to true, implying that it is not checked
-or unchecked.
-
-<Canvas of={MultiSelectStories.DialogControls} />
+<Source
+  code={`
+<MultiSelect
+  id="subject"
+  labelText="Subject Specialities"
```

Comment:
Is this code example mean to be what is rendered under "Component Props"? If so, then the title should be "MultiSelect" and you can add a "..." after the "Art History" object to show that you can have more objects in the array.